estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "estate",
+    "version": "1.0",
+    "depends": ["base"],
+    "author": "Kevin Gerard",
+    "category": "Category",
+    "description": """
+        This is a module to manage real estate properties
+    """,
+    "data": ["data/ir.model.access.csv"],
```

Comment:
```suggestion
    "data": ["security/ir.model.access.csv"],
```
By convention, security files must be under "security" folder.